lib/pbench/server/database/models/users.py

```diff
@@ -112,15 +127,11 @@ def delete(username):
             raise
 
     def is_admin(self):
-        # TODO: Add notion of Admin user
-        """this method would always return false for now until we add a notion of Admin user/group.
-        Once we know the admin credentials this method can check against those credentials to determine
-        whether the user is privileged to do more.
-
+        """This method checks whether the given user has an admin role.
         This can be extended to groups as well for example a user belonging to certain group has only those
         privileges that are assigned to the group.
         """
-        return False
+        return self.role == Roles.ADMIN
```

Comment:
Enums are supposed to be [compared by identity](https://docs.python.org/3/library/enum.html#comparisons) (i.e., using the `is` operator), but apparently the equality operator is defined.